static/css/tabzilla/tabzilla.css

```diff
@@ -0,0 +1,34 @@
+#tabzilla {
+  position: relative;
+  float: right;
+}
+#tabzilla a {
+  position: relative;
+  display: block;
+  width: 147px;
+  height: 37px;
+  text-indent: 120%;
+  white-space: nowrap;
+  overflow: hidden;
+  background-image: url('../../img/tabzilla/tabzilla-static.png');
+  background-repeat: no-repeat;
+  z-index: 2;
+}
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+  #tabzilla a {
+    background-image: url("../../img/tabzilla/tabzilla-static-high-res.png");
+    -webkit-background-size: 147px 37px;
+    background-size: 147px 37px;
+  }
+}
+#tabzilla:before {
+  position: absolute;
+  display: block;
+  left: 28px;
+  top: 0;
+  width: 88px;
+  height: 26px;
+  content: '';
+  background-color: #484848;
```

Comment:
Ah, good. Can you add that link to the comment?

But, that we should be using that specific color, so we should probably do that unless there's some reason it doesn't work. In which case, we should talk to someone from branding. They can get a bit tetchy (n.b., for any branding person who reads this: tetchy for completely reasonable reasons) about colors in brand marks.